### DISCUSSION TITLE
Reduce XP gained in battlegrounds to 1x for level 58+

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#4464] [Suggestions]
Considering the upcoming TBC release, I would suggest to either remove totally or at least reduce the experience gained in battlegrounds back to 1x.

possible would be also to make this change only for levels 58+

reasoning is that besides Eye of the Storm battleground that the value add for the server in the form of people running dungeons and doing quests to report issues is a lot higher.

**Comment by balleny** [comment]
Same essentially applies to Weekend XP.
for TBC Content, there shouldnt be x2 experience on the weekend

**Comment by Amrx2** [comment]
Hi balleny, you will also get this in an official announcement on the web page but upon the TBC release, it will work as follows:

RAF & Double EXP weekends will only work up to level 58 for the first bracket of TBC. It will be expanded upon once the level caps increase.
BG's we will also look at lowering them initially to begin with, we know a CTA AV weekend is quite literally the fastest way to level currently! 

Thanks for flagging this, have a good day.
Construct
